error.go

```diff
@@ -25,6 +25,9 @@ const (
 	ErrorMissingRecipient = "no recipients (to, cc, bcc) set"
 )
 
+// MaxPartErrors limits number of part parsing errors, 0 means no limit.
```

Comment:
Please set default to 0, and extend comment to note that errors after the limit are ignored, it does not cause parsing to fail.